config/passport.js

```diff
@@ -9,15 +9,17 @@ var User = require('../models/user');
 // passport needs ability to serialize and unserialize users out of session
 
 // used to serialize the user for the session
-passport.serializeUser(function (user, done) {
-    done(null, user.id);
+passport.serializeUser(function(user, done) {
+  console.log('serializing user: ', user.local.username);
+  done(null, user.id);
 });
 
 // used to deserialize the user
-passport.deserializeUser(function (id, done) {
-    User.findById(id, function (err, user) {
-        done(err, user);
-    });
+passport.deserializeUser(function(id, done) {
+  User.findById(id, function(err, user) {
+    console.log('deserializing user: ', user.local.username);
```

Comment:
are these console.logs absolutely necessary? try to keep clutter out of the console log and use it for more basic stuff such as "application initialized" and network errors. especially in the future when you use the console to debug stuff - having too many console.logs can work against development